src/app/monitoring-selection/selection-list/selection-list.component.ts

```diff
@@ -30,7 +30,8 @@ export class SelectionListComponent {
       .afterClosed()
       .subscribe(async (selectedService) => {
         if (!(selectedService === undefined)) {
-          this.monitoringSelectionService.addSelection(selectedService);
+          await this.monitoringSelectionService.addSelection(selectedService);
+          window.location.reload();
```

Comment:
Das halte ich für keine gute Lösung, da sollte man irgendwann nochmal nach der richtigen Ursache suchen und dann eine passende Lösung finden

Reply:
This looks like a workaround fix for a bug. This kind of fix should always have at least a TODO or FIXME comment describing the bug and the workaround applied to temporarily fix the bug. The comment is very helpful later to detect and fix technical debt like this. You could also directly add an issue to fix the workaround later.

Reply:
Hab den Bug gefixt, siehe neusten commit